conf/messages

```diff
@@ -713,4 +710,13 @@ prevalidateTrader.caption = Prevalidate a trader
 prevalidateTrader.start.title = Check if a trader can receive excise goods
 prevalidateTrader.start.h1 = Check if a trader can receive excise goods
 prevalidateTrader.start.p1 = You can use this tool to check if the consignee is able to receive your goods. You will need their trader ID, such as an excise registration number (ERN) or temporary authorisation number.
-prevalidateTrader.start.button =
+
+# Prevalidate trader - consignee trader identification page
+prevalidateTrader.consigneeTraderIdentification.title = What is the consignee’s trader ID?
+prevalidateTrader.consigneeTraderIdentification.h1 = What is the consignee’s trader ID?
+prevalidateTrader.consigneeTraderIdentification.p1 = This could be an excise registration number (ERN) or a temporary authorisation reference for temporary consignees, sometimes known as a Temporary Consignment Authorisation (TCA) number.
```

Comment:
I couldn't see a full stop in the copydeck, assume thats a typo in the copydeck

Reply:
I've asked UCD, will update the PR or copydeck with the correct version

Reply:
Full stop was missing in the copydeck, p1 is correct here.